packages/rocketchat-reactions/setReaction.js

```diff
@@ -39,8 +39,6 @@ Meteor.methods({
 			return false;
 		}
 
-		reaction = `:${ reaction.replace(/:/g, '') }:`;
```

Comment:
Heh I knew we had this code in place somewhere, but I just didn't open my eyes enough to see that it was a few lines below.